From e7b9e246430753fb31a39bc64d0f65ea9a716dca Mon Sep 17 00:00:00 2001
From: Jiri Moskovcak <jmoskovc@redhat.com>
Date: Sun, 31 Jul 2011 14:19:02 +0200
Subject: [PATCH 2/4] don't reload event configuration when dump_dir changes

- reloading config erases the current session configuration and
  if keyring is not available it leaves user credentials empty
  which makes gui complain about wrong plugin configuration
---
 src/gui-wizard-gtk/main.c |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/src/gui-wizard-gtk/main.c b/src/gui-wizard-gtk/main.c
index 344d109..da32998 100644
--- a/src/gui-wizard-gtk/main.c
+++ b/src/gui-wizard-gtk/main.c
@@ -74,10 +74,6 @@ void reload_problem_data_from_dump_dir(void)
     }
     g_cd = new_cd;
 
-    /* Load /etc/abrt/events/foo.{conf,xml} stuff */
-    load_event_config_data();
-    load_event_config_data_from_keyring();
-//TODO: Load ~/.abrt/events/foo.conf?
 }
 
 int main(int argc, char **argv)
-- 
1.7.6

